c4d_migrate/includes/ImportBaseGroup.inc

```diff
@@ -43,6 +43,8 @@ abstract class C4dMigrateImportBaseGroup extends C4dMigrateImportBaseNode {
     $this->addFieldMapping('group_access', 'og_private');
 
     // Map the group status.
+    $this->removeFieldMapping('status', 'status');
+    $this->addFieldMapping('status', '_status');
```

Comment:
I think you can use ``$this->addFieldMapping('status')->defaultValue(NODE_PUBLISHED);`` instead